products.d/ALP-Dolomite.yaml

```diff
@@ -1,8 +1,14 @@
 id: ALP-Dolomite
 name: SUSE ALP Dolomite
+# ------------------------------------------------------------------------------
+# WARNING: When changing the product description delete the translations located
+# at the at translations/description key below to avoid using obsolete
+# translations!!
+# ------------------------------------------------------------------------------
 description: 'SUSE ALP Dolomite is a minimum immutable OS core, focused on
   security to provide the bare minimum to run workloads and services as
   containers or virtual machines.'
+translations:
```

Comment:
Explicitly adding an empty `translations` key. Otherwise the script would add it at the end of the file and it is better to have the translations close to the original text.